backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestResultUpload.java

```diff
@@ -34,6 +35,7 @@ public class TestResultUpload extends AuditedEntity {
 
   @ManyToOne(optional = false, fetch = FetchType.LAZY)
   @JoinColumn(name = "org_id")
+  @JsonIgnore
   private Organization organization;
```

Comment:
Ignoring this since we don't want to return it as part of the response object when uploading results